client/src/api/axios.ts

```diff
@@ -14,4 +14,14 @@ const BaseClient: AxiosInstance = axios.create({
   timeout: 1000,
 });
 
-export { AuthClient, BaseClient };
+async function Search(searchInput:any) {
+  await AuthClient.get(`/members/search/${searchInput}`)
+  .then(response=>{
+    return response.data;
+  })
+  .catch(error=>{
+    console.error(error);
+  })
+}

```

Comment:
```suggestion
  return await AuthClient.get(`/members/search/${searchInput}`)
}

```

and handle it with try and catch when you call it